crates/cubecl-macros/Cargo.toml

```diff
@@ -21,7 +21,20 @@ default = []
 std = []
 
 [dependencies]
+darling = { workspace = true }
+derive-new = { workspace = true }
+ident_case = { workspace = true }
+prettyplease = "0.2"
 proc-macro2 = { workspace = true }
 quote = { workspace = true }
 syn = { workspace = true }
-derive-new = { workspace = true }
+
+cubecl-common = { path = "../cubecl-common", version = "0.2", default-features = false }
+
+[dev-dependencies]
+compiletest_rs = { version = "0.11", features = ["tmp"] }
+cubecl-core = { path = "../cubecl-core", version = "0.2", default-features = false }
+cubecl-cuda = { path = "../cubecl-cuda", version = "0.2", default-features = false }
+cubecl-linalg = { path = "../cubecl-linalg", version = "0.2", default-features = false }
+cubecl-wgpu = { path = "../cubecl-wgpu", version = "0.2", default-features = false }
```

Comment:
Why are they needed?

Reply:
For the codegen comparison tests. Could move them but I put them here because they're testing the macro, not the backends in question. Or maybe just use the testgen macro thing.